guides/common/modules/proc_changing-the-environment-of-a-host.adoc

```diff
@@ -6,7 +6,7 @@ Use this procedure to change the environment of a host.
 .Procedure
 . In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
 . Click the name of the host you want to modify.
-. Click the vertical ellipsis in the *Content view details* card and select *Edit content view assignment*.
+. Click the icon in the *Content view environment* card and select *Edit content view environments*.
```

Comment:
```suggestion
. Click the options icon on the *Content view environment* card and select *Edit content view environments*.
```

Reply:
It looks like we call this the "options icon" (no markup or capitalization if there is no tooltip or label for the icon). 

I suggested changing the preposition because "on the card" sounds more idiomatic to me than "in the card". Your call.